x/ccv/consumer/keeper/relay.go

```diff
@@ -196,13 +196,17 @@ func (k Keeper) SendPackets(ctx sdk.Context) {
 		)
 		if err != nil {
 			if clienttypes.ErrClientNotActive.Is(err) {
-				k.Logger(ctx).Debug("IBC client is inactive, packet remains in queue", "type", p.Type.String())
+				// IBC client is expired!
 				// leave the packet data stored to be sent once the client is upgraded
+				k.Logger(ctx).Debug("IBC client is expired, cannot send IBC packet; leaving packet data stored:", "type", p.Type.String())
```

Comment:
Can we promote this log line to `Info`?

It seems useful to know about inactive clients without turning on debug mode.

Reply:
It may be a good idea. It was like this before though that's why I decided not to change it.